src/main/java/xyz/vegaone/easytrackingv2/mapper/OrganizationMapper.java

```diff
@@ -0,0 +1,93 @@
+package xyz.vegaone.easytrackingv2.mapper;
+
+import org.mapstruct.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.CollectionUtils;
+import xyz.vegaone.easytrackingv2.domain.OrganizationEntity;
+import xyz.vegaone.easytrackingv2.domain.ProjectEntity;
+import xyz.vegaone.easytrackingv2.domain.UserEntity;
+import xyz.vegaone.easytrackingv2.dto.Organization;
+import xyz.vegaone.easytrackingv2.dto.Project;
+import xyz.vegaone.easytrackingv2.dto.User;
+
+import java.util.List;
+
+@Mapper(componentModel = "Spring")
+public abstract class OrganizationMapper {
+
+    @Autowired
+    private ProjectMapper projectMapper;
+
+    @Autowired
+    private UserMapper userMapper;
+
+    @Mappings({
+            @Mapping(target = "userList", ignore = true),
+            @Mapping(target = "projectList", ignore = true)
+    })
+    public abstract Organization domainToDto(OrganizationEntity organizationEntity);
+
+    @Mappings({
+            @Mapping(target = "userList", ignore = true),
+            @Mapping(target = "projectList", ignore = true)
+    })
+    public abstract OrganizationEntity dtoToDomain(Organization organization);
+
+    public abstract List<Organization> domainToDtoList(List<OrganizationEntity> organizationEntityList);
+
+    public abstract List<OrganizationEntity> dtoToDomainList(List<Organization> organizationList);
+
+    @AfterMapping
+    void addUserProjectToDto(OrganizationEntity organizationEntity, @MappingTarget Organization organization) {
```

Comment:
The name of the method implies that only the user is added, but the scope is greater than that. Could you please rename it to, hm, `addIgnoredFiedlsToDto`